app/Http/Controllers/Api/FishesController.php

```diff
@@ -6,10 +6,19 @@
 use App\Models\Fishes;
 use App\Models\HasFish;
 use App\Models\User;
+use App\Services\PeriodService;
 use Illuminate\Http\Request;
 
 class FishesController extends Controller
 {
+    /** @var PeriodService $periodService */
+    private $periodService;
```

Comment:
tu peux directement typer la variable
```suggestion
    private PeriodService $periodService;
```